lib/graphql/input_object_type.rb

```diff
@@ -75,16 +75,14 @@ def coerce_non_null_input(value)
 
       arguments.each do |input_key, input_field_defn|
         field_value = value[input_key]
-        field_value = input_field_defn.type.coerce_input(field_value)
 
-        # Try getting the default value
-        if field_value.nil?
```

Comment:
> So, I think I'd say, we were probably missing a test for the old behavior. But that said, maybe there's an argument to be made that `field` and `field(arg: null)` should be treated differently? (Or `{ }` and `{ arg: nil }`)

Yeah, they absolutely should be treated differently. One is omitted and one is explicitly `null`. This PR introduces  that difference. 😀 `field(arg: null)` wasn't a thing before.